src/goto-instrument/cover_instrument_branch.cpp

```diff
@@ -21,7 +22,9 @@ void cover_branch_instrumentert::instrument(
   if(is_non_cover_assertion(i_it))
     i_it->turn_into_skip();
 
-  if(i_it == goto_program.instructions.begin())
+  if(
+    i_it == goto_program.instructions.begin() &&
```

Comment:
:bulb:  Unrelated to this PR, just for code clarity
 If I understand correctly, `i_it == goto_program.instructions.begin()` means "is entry point", so why not have 
`bool is_entry_point = i_it == goto_program.instructions.begin()` and then use that in the condition?